sources/packages/web/src/views/aest/AppAEST.vue

```diff
@@ -13,21 +13,32 @@
         "
         ><v-icon icon="fa:fa fa-edit"></v-icon>Create institution</v-btn
       >
-      <v-btn
-        v-if="isAuthenticated"
-        class="mr-5"
-        icon="fa:fa fa-user"
-        variant="outlined"
-        elevation="1"
-        color="grey"
-        @click="toggleUserMenu"
-      ></v-btn>
-      <Menu
-        v-if="isAuthenticated"
-        ref="userOptionsMenuRef"
-        :model="userMenuItems"
-        :popup="true"
-      />
+      <v-menu v-if="isAuthenticated">
+        <template v-slot:activator="{ props }">
+          <v-btn
+            class="mr-5"
+            icon="fa:fa fa-user"
+            variant="outlined"
+            elevation="1"
+            color="grey"
+            v-bind="props"
+          ></v-btn>
+        </template>
+        <v-list>
+          <template v-for="(item, index) in menuItems" :key="index">
```

Comment:
For such small menus I would prefer to avoid the `for` and checks like `v-if="index < menuItems.length - 1"`.

Reply:
mm. I added this logic with the assumption that this menu will grow in the future. I am okay to remove v-for